app/react-native-server/src/client/manager/provider.js

```diff
@@ -76,8 +62,7 @@ export default class ReactProvider extends Provider {
   handleAPI(api) {
     addons.loadAddons(api);
 
-    api.onStory((kind, story) => {
-      this.selection = { kind, story };
+    api.onStory(() => {
```

Comment:
`api.onStory` should at some point be changed to `api.on`

I understand onStory is kinda convenient, but I don't want to add a method for every event.